miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -264,13 +278,13 @@ abstract class MiniAppMessageBridge {
             postError(callbackId, "${ErrorBridgeMessage.ERR_SHOW_AD} ${ErrorBridgeMessage.ERR_NO_SUPPORT_HOSTAPP}")
     }
 
-    /** Return a value to mini app. **/
-    internal fun postValue(callbackId: String, value: String) {
+    /** Emit a value to mini app. **/
+    override fun postValue(callbackId: String, value: String) {
```

Comment:
`postValue` and `postError` are still public. The `MiniAppMessageBridge` is a public class that's why it's still exposed even we declare the listener interface as internal. 
Here is [suggestion](https://github.com/khairul-alam-licon/android-miniapp/blob/bb1083fc3263c6878592d64adb69f27b2a140665/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/BridgeExecutor.kt) from my PR.

Reply:
Ah, I didn't noticed it. Updated according to your suggestion.